backend/canisters/group_index/CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- Integrate Communities ([#3656](https://github.com/open-ic/open-chat/pull/3656)), ([#3657](https://github.com/open-ic/open-chat/pull/3657)), ([#3687](https://github.com/open-ic/open-chat/pull/3687))
+- Integrate Communities ([#3656](https://github.com/open-ic/open-chat/pull/3656)), ([#3657](https://github.com/open-ic/open-chat/pull/3657)), ([#3687](https://github.com/open-ic/open-chat/pull/3687)), ([#3759](https://github.com/open-ic/open-chat/pull/3759)) 
```

Comment:
This should be in the unreleased section

Reply:
Oh dear so careless!